setup.py

```diff
@@ -45,6 +45,7 @@
     'flux': ['pyyaml', 'cffi', 'jsonschema'],
     'proxystore': ['proxystore'],
     'radical-pilot': ['radical.pilot==1.90', 'radical.utils==1.90'],
+    'globus_compute': ['globus_compute_sdk>=2.27.1'],
     # Disabling psi-j since github direct links are not allowed by pypi
```

Comment:
It's been a minute since we started this PR; might increase this to the most up-to-date as of ~now (2.34.0)

Reply:
Fixed.